code/VersionedGridFieldDetailForm.php

```diff
@@ -136,18 +171,19 @@ public function getCMSActions() {
 					->setUseButtonTag(true)->addExtraClass('ss-ui-action-constructive')->setAttribute('data-icon', 'accept')->setDisabled(true)
 			);
 		}
-
 		// This is a bit hacky, however from what I understand ModelAdmin / GridField dont use the SilverStripe navigator, this will do for now just fine.
-		// ensure link method is defined & non-null before allowing preview
-		if ($this->canPreview() && method_exists($this->record, 'Link') && $this->record->Link()) {
-			$actions->push(
-				LiteralField::create("preview",
-					sprintf("<a href=\"%s\" class=\"ss-ui-button\" data-icon=\"preview\" target=\"_blank\">%s &raquo;</a>",
-						$this->record->Link()."?stage=Stage",
-						_t('LeftAndMain.PreviewButton', 'Preview')
```

Comment:
Unless we're doing a full boolean evaluation the old and the new versions (above, below) don't seem functionally different. But I'm sure there's a reason ...